pkg/ui/app/containers/clusterViz.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import {runVisualization, viewWidth, viewHeight} from "../js/sim/index";
+
+/**
+ * Renders the layout of the ClusterViz page. "ClusterViz" is an experimental
+ * visual map of the cluster, expressed as a directive 
```

Comment:
Is this comment complete? Or does it just need end punctuation? I don't know what a directive is in this context...